spec/Section 3 -- Type System.md

```diff
@@ -357,9 +358,15 @@ excluding fragments for which the type does not apply and fields or
 fragments that are skipped via `@skip` or `@include` directives. This ordering
 is correctly produced when using the {CollectFields()} algorithm.
 
-Response formats which support ordered maps (such as JSON) must maintain this
-ordering. Response formats which do not support ordered maps may disregard
-this ordering.
+Response formats and runtime environments capable of representing ordered maps
+should maintain this ordering when possible. Serialization formats which can
```

Comment:
I might remove "when possible" here; if it's capable of representing ordered maps, then it is possible!